<!-- -->
<launch>
	<arg name="limit_euc" default="6.0"/>
	<arg name="permission_angle" default="30"/>

	<node pkg="waypoint_follower" type="position_checker" name="position_checker" output="log">
		<param name="limit_euc" value="$(arg limit_euc)"/>
		<param name="permission_angle" value="$(arg permission_angle)"/>
	</node>
</launch>
